.gitignore

```diff
@@ -12,7 +12,6 @@ x64/
 x86/
 build/
 bld/
-[Bb]in/
```

Comment:
This was conflicting with the rust crates bin directory, I've removed it because I don't think it's applicable anywhere else.